mozilla_workgroup/variables.tf

```
@@ -7,3 +7,32 @@ variable "ids" {
     error_message = "Bad workgroup identifier format, must match workgroup:WORKGROUP[/SUBGROUP] or subgroup:SUBGROUP."
   }
 }
+
+variable "roles" {
+  type        = map(string)
+  description = "List of roles to generate bigquery acls for"
+  default = {
+    metadata_viewer = "roles/bigquery.metadataViewer"
+    read            = "READER"
+    write           = "WRITER"
+  }
```

```suggestion
  default = {}
```
I am of the opinion to change this to an empty map, and put a comment of potential options.

---

please take a look at the comment I added and let me know .. thanks

---

LGTM